layouts/shortcodes/blocks/feature.html

```diff
@@ -2,10 +2,17 @@
 {{ $url_text := .Get "url_text" -}}
 <div class="col-lg-4 mb-5 mb-lg-0 text-center">
 <div class="mb-4 h1">
+{{ with .Get "url" }}<a class="text-white" target="_blank" href="{{ . }}">
```

Comment:
You are using two separate blocks (`{{ with }}` +  `{{ if }}`) here. Why don't we assign the inner content to a variable? Then we can have one single `{{ with }} ... {{ else }}` block here, improving the readability of the code.

Reply:
Resolved